src/test/clojure/xtdb/node_test.clj

```diff
@@ -8,17 +8,14 @@
 
 (t/deftest test-multi-value-insert-423
   (letfn [(expected [tt]
-            {[(util/->zdt #inst "2024-01-01") (util/->zdt util/end-of-time), tt tt]
-             "Happy 2024!",
-
-             ;; weird? zero-width system-time so won't normally show up
-             [(util/->zdt #inst "2024-01-01") (util/->zdt #inst "2026-01-01"), tt tt]
+            {;; FIXME once a decision stands on zero-width intervals #2257
```

Comment:
if your change happens to remove zero-width intervals, that's great - and feel free to tick off #2257